controller-server/src/main/java/com/yahoo/vespa/hosted/controller/tenant/Contact.java

```diff
@@ -46,6 +51,39 @@ public URI issueTrackerUrl() {
         return persons;
     }
 
+    public Slime toSlime() {
```

Comment:
Serialization logic does not really belong in the data model class. Please move this to `ContactInfoHandler`.